src/app/ubs-admin/components/ubs-admin-employee/employee-form/employee-form.component.html

```diff
@@ -0,0 +1,61 @@
+<h1>Додати працівника</h1>
+<form [formGroup]="employeeForm" (ngSubmit)="submit()">
+  <div class="form-group">
+    <h6>*Обов'язкові поля</h6>
+  </div>
+  <div class="form-group">
+    <label for="name">Ім'я*</label>
+    <input formControlName="firstName" placeholder="Placeholder" />
```

Comment:
placeholder="Placeholder" ? or placeholder="First name"

Reply:
In mockup on Figma was placeholder="Placeholder", but fixed it